### PR TITLE
Adding Onboarding E2E to GH

### PR DESCRIPTION
**Pull request by diogormendes**
3 times a few times and everything looks OK

Screen Shot 2022-04-01 at 11 47 13
Screen Shot 2022-04-01 at 11 47 21
Screen Shot 2022-04-01 at 11 47 31

### CODE REVIEW

File: .github/workflows/uberjar.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Build + Percy Test
+name: Build Uberjar + Docker
```
[the remark this answers is not in the document]
Comment (diogormendes) [reply .github/workflows/uberjar.yml:1]: Yeah, just fixed the name to a one that is more correct. (typo)

[the remark this answers is not in the document]
Comment (diogormendes) [reply .github/workflows/uberjar.yml:1]: We are not running Percy anymore as you can see here https://github.com/metabase/metabase/pull/21383

[the remark this answers is not in the document]
Comment (ariya) [reply .github/workflows/uberjar.yml:1]: then it should be a separate PR

**Comment by nemanjaglumac** [comment]
Let's rebase this to the latest `master`, just to make sure things still work fine and then it will be good to merge.

**Review by flamber — APPROVED** [review]

**Pull request merged**